src/views/ModelPlan/TaskList/ITSolutions/AddSolution/index.tsx

```diff
@@ -322,16 +322,23 @@ const AddSolution = () => {
                                 })}
                             </Field>
 
-                            {values.key ===
-                              OperationalSolutionKey.OTHER_NEW_PROCESS &&
+                            {(values.key ===
+                              OperationalSolutionKey.CONTRACTOR ||
+                              values.key ===
+                                OperationalSolutionKey.CROSS_MODEL_CONTRACT ||
+                              values.key ===
+                                OperationalSolutionKey.EXISTING_CMS_DATA_AND_PROCESS ||
+                              values.key ===
+                                OperationalSolutionKey.OTHER_NEW_PROCESS) &&
```

Comment:
I think there is a solution missing from here.  Also I think we can probably condense this to have an entity containing all the `other` enums and perform `.includes(value.key)`